aws/sdk/examples/snowball/src/bin/describe-addresses.rs

```diff
@@ -3,29 +3,50 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
-use aws_sdk_snowball::{Config, Region};
-use aws_types::region;
+use aws_sdk_snowball::{Client, Config, Error, Region, PKG_VERSION};
+use aws_types::region::{self, ProvideRegion};
 use structopt::StructOpt;
 
 #[derive(Debug, StructOpt)]
 struct Opt {
-    /// The region
+    /// The AWS Region.
     #[structopt(short, long)]
     region: Option<String>,
+
+    /// Whether to display additional information.
+    #[structopt(short, long)]
+    verbose: bool,
 }
 
+/// Lists your AWS Snowball addresses.
+/// # Arguments
+///
+/// * `[-r REGION]` - The Region in which the client is created.
+///    If not supplied, uses the value of the **AWS_REGION** environment variable.
+///    If the environment variable is not set, defaults to **us-west-2**.
+/// * `[-v]` - Whether to display additional information.
 #[tokio::main]
-async fn main() -> Result<(), aws_sdk_snowball::Error> {
+async fn main() -> Result<(), Error> {
     tracing_subscriber::fmt::init();
 
-    let Opt { region } = Opt::from_args();
+    let Opt { region, verbose } = Opt::from_args();
 
     let region_provider = region::ChainProvider::first_try(region.map(Region::new))
         .or_default_provider()
-        .or_else(Region::new("us-east-1"));
+        .or_else(Region::new("us-west-2"));
+
+    println!();
+
+    if verbose {
+        println!("Snowball version: {}", PKG_VERSION);
+        println!(
+            "Region:                {}",
+            region_provider.region().unwrap().as_ref()
```

Comment:
`as_ref()` not needed here, I think

Reply:
It's the only way I can get just the region value, without quote marks.